src/python/arcor2_arserver/lock.py

```diff
@@ -266,13 +271,14 @@ def get_root_id(self, obj_id: str) -> str:
             obj = self.project.action_point(obj_id)
             return obj.parent if obj.parent else obj_id
 
-        elif obj_id in [self.SCENE_NAME, self.PROJECT_NAME]:
+        # TODO find out, how enum works
+        elif obj_id in [self.SpecialValues.SCENE_NAME, self.SpecialValues.PROJECT_NAME]:
```

Comment:
Je to úplně detail, ale osobně v podobných případech používám raději tuple, který je immutable. Prakticky v tom asi není rozdíl.